pkg/minikube/cluster/mount_test.go

```diff
@@ -23,19 +23,20 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
-type mockMountHost struct {
+// MockMountHost is for testing
+type MockMountHost struct {
 	cmds []string
 	T    *testing.T
 }
 
-func NewMockMountHost(t *testing.T) *mockMountHost {
-	return &mockMountHost{
+func NewMockMountHost(t *testing.T) *MockMountHost {
```

Comment:
Make this function name private (newMockMountHost) - then the underlying struct can be private too.

There's no need to pollute the public namespace with this minimal implementation.

Reply:
Right